.github/workflows/continuous-integration.yml

```diff
@@ -12,26 +12,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Check out wala/ML sources
-      uses: actions/checkout@v3
-    - name: Set up JDK 17
-      uses: actions/setup-java@v3
+      uses: actions/checkout@v4
+    - name: Set up JDK 24
```

Comment:
@msridhar wanted to stick with an LTS version, either 21 or wait for the next one. Thoughts?